### ISSUE TITLE
[User flows and navigating] A user flow doesn't provide content or navigation to help users understand where to go or what to do next. (03.02.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#68270]
#### VFS team name
Authenticated Experience

#### VFS product name
My VA

#### VFS feature name
Dot indicator

#### Point of Contact/Reviewers
Erin White - @erinrwhite - IA
Brian Smolensky - @BSmolensky - Content

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
A user flow doesn't provide content or navigation to help users understand where to go or what to do next.

### Issue Details
When a user clicks on the "Go to my inbox" from My VA and is sent to MHV secure messaging, there is little visual information towards the top of the secure messages page that indicates why there was a notification - **except** the red text that matches the color of the notification dot, urging Veterans to call 911 if experiencing a medical emergency. After scrolling down, new messages are indicated in bold. This is not launch-blocking, and might be beyond this team's purview, but could be confusing for users.

>Secure Messaging: To arrive at this page, the user clicked on a link that said "go to inbox" The user should reasonably expect to arrive at their inbox once they click that link. Instead, they find a page that requires quite a bit of scrolling before they can find their inbox. This is unnecesarily confusing. Please clean up this page to allow the user to arrive at their desired destination sooner. 

### VA.gov Experience Standard
[Category Number 03, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Possible solutions would involve changes to the MHV secure messaging page. I understand if this is not within the team's span of control or influence.

- Reduce the amount of content at the top of the MHV secure messaging page so that users don't have to search for the notification information.
- Change the color on the "PLEASE CALL 911 if you have a medical emergency or need urgent medical assistance." to be different from the red notification dot's color.
- Consider adding yet another visual indicator to the list of new messages showing unread status - perhaps a light background color on the table cell in addition to the bolded text.

Please clean up this page. 

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by ajakabcin** [comment]
@shiragoodman it's my understanding that work on legacy secure messaging is not going to be prioritized over work on the new secure messaging tool. We've asked the secure messaging team if we're able to link to the new tool as that would resolve this issue since there are indicators about unread/new messages to align with the dot on My VA, but the agreed plan with MHV stakeholders is to not provide direct access to the tool until phase 2, so I don't see this being resolved any time soon. ([More details in this thread](https://dsva.slack.com/archives/C03ECSBGSKX/p1698162493809239))

**Comment by shiragoodman** [comment]
thanks @ajakabcin. I am going to remove your team label and product label from this ticket.

Question you might know the answer to... why does MyVA even link out to the legacy MHV secure messaging in the first place? Has it always or was this a decision made a while back?

**Comment by ajakabcin** [comment]
Yup! this has always been the practice on My VA for health care tools/resources. Prior to the migration work, that was the only place to send users to complete health care tasks.
